lib/src/fields/form_builder_segmented_control.dart

```diff
@@ -117,5 +117,5 @@ class FormBuilderSegmentedControl<T> extends FormBuilderField<T> {
       _FormBuilderSegmentedControlState();
```

Comment:
@AlBlanc This must be `_FormBuilderSegmentedControlState<T>();`. We must use generic type when creating instance of State classes. It is needed to double check all such occurrences, because it will lead to runtime exceptions.